### PR TITLE
Minor updates for Fleet/Device list pages

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/app/components/Fleet/SourceUrlList.tsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 
-import { Button } from '@patternfly/react-core';
+import { Button, Stack, StackItem } from '@patternfly/react-core';
 
 const SourceUrlList = ({ sourceUrls }: { sourceUrls: string[] }) => {
```
Comment (celdrake) [inline src/app/components/Fleet/SourceUrlList.tsx:5]: From `getSourceUrls`, sources are now references to a repository "name".
So we should have changed the link as if it's a URL, to instead being a link to a repository. 

Could you change it in this PR?

**Review by celdrake — COMMENTED** [review]
There's an incosistency in `MassDeleteFleetModal`, as there we're keeping the previous design (not showing the new git icon and showing the "managed by" column).

Also, I think it would be better to display separately the list of fleets which would be deleted and those that wouldn't. And we should also handle the case when none of the fleets can be deleted.

For this ticket, I think adding the icon to the managed repositories is enough, and we could create a new ticket to address these inconsistencies. WDYT?

File: src/app/components/Fleet/SourceUrlList.tsx (hunk 1 above)
Comment (rawagner) [reply src/app/components/Fleet/SourceUrlList.tsx]: ok, this is fixed now

**Comment by rawagner** [comment]
I've actually kept the Managed By column on purpose as we have more space in the mass delete modal and in details page. We can discuss this with UX and remove in a followup

File: src/app/components/Fleet/RepositorySourceList.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+const RepositorySourceList = ({ repositorySources }: { repositorySources: string[] }) => {
+  const { t } = useTranslation();
+  const { get } = useFetch();
+
+  const [repoLinks, setRepoLinks] = React.useState<string[]>();
+  const [errors, setErrors] = React.useState<string[]>();
+
+  const sources = useArrayEq(repositorySources);
```
Comment (celdrake) [inline src/app/components/Fleet/RepositorySourceList.tsx:27]: I thought you would set links to the repository names, but also checking that they are valid references is much better!

**Review by celdrake — APPROVED** [review]

**Pull request merged**